internal/gitinterface/commit.go

```diff
@@ -116,8 +116,13 @@ func CreateCommitObject(gitConfig *config.Config, treeHash plumbing.Hash, parent
 		TreeHash:  treeHash,
 		Message:   message,
 	}
-	if !parentHash.IsZero() {
-		commit.ParentHashes = []plumbing.Hash{parentHash}
+	if len(parentHashes) > 0 {
```

Comment:
```suggestion

	if len(parentHashes) > 0 {
```

A new line here would be nice now to separate this block out.

Reply:
@neilnaveen I've marked the other one as resolved. Can you add this newline and squash, I can then merge? Thanks!